Examples/Tests/Maxwell_Hybrid_QED/analysis_Maxwell_QED_Hybrid.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+chmod a+x analysis_Maxwell_QED_hyrbid.py
```

Comment:
ah no, sorry.
Do not add the line here but execute it on this python file to change its meta data please :)

Reply:
ahhhh. I thank you!

Reply:
But I may still not understand.

Reply:
If you run `ls -l Examples/Tests/Maxwell_Hybrid_QED/analysis_Maxwell_QED_Hybrid.py` on your terminal it will show you [the permissions](https://warwick.ac.uk/fac/sci/csc/students/modules/cy900/current/session1/concepts/permissions/), size and modification time of a file.

`chmod a+x ...` ("change file mode bits", see `man chmod`) changes the permissions so you can just e**x**ecute a file by its name: `cd ./Examples/Tests/Maxwell_Hybrid_QED; ./analysis_Maxwell_QED_Hybrid.py`. The first line indicates how to execute this text file - as python3 script :)

Reply:
ohhhhh. I think I get it now. Thank you!!